javascript/exocom-lint/features/lint.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Linting Exosphere applications
+
+  As an Exosphere developer
+  I want to have an easy way to confirm all messages being sent are received somewhere
+  So that I know that all my messages are handled.
+
+  Rules:
+  - run "exo lint" in the directory of your application to lint it
+  - if a sent message is not received anywhere, it prints an error message
+  - if a received message it not sent anywhere, it prints an error message
+
+  Scenario: functioning application
+    Given I am in the directory of an application with the services:
+      | NAME  | SENDS                      | RECEIVES                   |
```

Comment:
If these `sends` and `receives` values are coming from the service.yml, I believe that they list internal message names, not public, so it might be prudent to write a test for that case.